core/performance_test/peakflops/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# PeakFlops uses x86 asm volatile
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "([xX]86|[aA][mM][dD]64)")
+
+    set(GTEST_SOURCE_DIR ${CMAKE_SOURCE_DIR}/gtest)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGTEST_HAS_PTHREAD=0")
+
+    include_directories(${GTEST_SOURCE_DIR})
+
+    include_directories(${CMAKE_CURRENT_BINARY_DIR})
+    include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+    foreach(_test 01_cpp_simple 02_kokkos_simple_view 03_Cabana_peakflops)
+        # TODO: tidy unit_test_main.cpp path
```

Comment:
Could you use `${GTEST_SOURCE_DIR}` for that?

Reply:
Do you mean

`foreach(_test ${GTEST_SOURCE_DIR})`? If so, yes we could, be I don't really love the globbing over a simple small list